### DISCUSSION TITLE
How can I move to the next / previous search results?

### DISCUSSION THREAD

**Discussion by Archie-2021** [tadashi-aikawa/obsidian-another-quick-switcher#176] [General]
Does anyone here knows how I can navigate between pages of search results in Another Quick Switcher Menus. There is no mention of it in documentation and settings? I know it is a very basic matter but I can't recall what it was.

**Comment by tadashi-aikawa** [comment]
@Archie-2021
Sorry for late answer 🙇 

Are you referring to the preview?

![image](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/assets/9500018/054cd5ef-d5d5-4641-bc1d-aa78848eb2c5)

**Reply by Archie-2021** [reply]
> @Archie-2021 Sorry for late answer bow
> 
> Are you referring to the preview?
> 
> ![image](https://user-images.githubusercontent.com/9500018/244848982-054cd5ef-d5d5-4641-bc1d-aa78848eb2c5.png)

No problem my friend, 
Now that I read it again it is very unclear so I should explained it better.

I am talking about when we search for something and it is in multiple pages (like showing 50 results out of total 500) , how we can navigate to previous and next page of results?

**Reply by tadashi-aikawa** [reply]
@Archie-2021 
Thank you for your clarification. I understand, but currently, there's no such feature.

The reason for limiting the number of displayed items is solely for performance (not for searching but displaying). You're right, it might be necessary to have a feature that displays the next/previous N items.

If you need it, please create an issue, then I'll try implementing it👍
